frontend/pages/home/hooks/tokenHook.tsx

```diff
@@ -24,8 +24,11 @@ export const TokenHook = (asset: Asset | undefined) => {
     address: "",
```

Comment:
The same default for what it looks. maybe a factory function that creates de default value?